src/frontend/src/handler/create_source.rs

```diff
@@ -1215,6 +1250,56 @@ pub(super) fn check_nexmark_schema(
     Ok(())
 }
 
+fn validate_dynamodb_source(
```

Comment:
Better to enhance the error messages to be more user-friendly like https://github.com/risingwavelabs/risingwave/pull/17397/files#diff-7e4135c7d009775db1ad2f1559fea1061b1b83860a3cc947feaede9b833edfe6

Reply:
Updated.